src/lib/logger.ts

```diff
@@ -46,6 +46,7 @@ interface LogMessage {
   event: string;
   method: string;
   parameters: any;
+  activityUrl: string | undefined;
```

Comment:
```suggestion
  activityUrl?: string;
```
and I would move it up to line 29, between `application` and `run_remote_endpoint`, because it's really more related to those items than to the CLUE-specific items below it.